src/components/cart/CartToggle/cart-toggle.scss

```diff
@@ -0,0 +1,33 @@
+.cart-toggle[data-store-button] {
+  position: relative;
+  background-color: transparent;
+  border-width: 0;
+  cursor: pointer;
+
+  svg {
+    width: var(--space-5);
+    height: var(--space-5);
+  }
```

Comment:
You can use `<ShoppingCart size={32} />` instead 🙃

Reply:
I had done that, but I preferred to leave everything on CSS. Is it better to put in js?